esmvaltool/config-user.yml

```diff
@@ -19,6 +19,8 @@ output_file_type: pdf
 output_dir: ./esmvaltool_output
 # Save intermediary cubes in the preprocessor true/[false]
 save_intermediary_cubes: false
+# Run tasks in parallel true/[false]
+parallel: true
```

Comment:
It looks like the default is not `False`: I tried it with my old config-user.yml (without `parallel` specification) and it ran parallel anyway.